{{cookiecutter.repo_name}}/frontend/js/links.ts

```diff
@@ -0,0 +1,40 @@
+function isExternalLink(link: HTMLAnchorElement) {
```

Comment:
Maybe consider something like this: https://github.com/mrded/is-url-external/blob/master/index.js

Reply:
Although, below, I see you read all the A tags on the page. You might consider deleting them as well. Just seems more full proof to do it using the host name and I think the small performance penalty would be worth it.